docs/source/examples/shapes_images_positions.rst

```diff
@@ -43,4 +81,154 @@ Shapes, Images and Positions
             self.add(img_mobject)
 
 
-Note: Here can come the UnitDot Example.
\ No newline at end of file
+.. manim:: ArcShapeExample
+    :save_last_frame:
+    
+    class ArcShapeExample(Scene):
+        def construct(self):
+            colors = [DARK_BLUE , DARK_BROWN, BLUE_E, BLUE_D, BLUE_A, TEAL_B, GREEN_B, YELLOW_E]
+            radius = [1+rad*0.1 for rad in range(len(colors))]
+            
+            circles_group=VGroup()
+            
+            # zip(radius,color) makes the iterator [(radius[i],color[i]) for i in range(radius)]
+            circles_group.add( *[Circle(radius=rad,stroke_width=10, color = col)
+                         for rad,col in zip(radius,colors)])
+            
+            self.add(circles_group)
+            self.wait(2)
+
+
+.. manim:: InterpolationExample
+    :save_last_frame:
+    
+    class InterpolationExample(Scene):
+        def construct(self):
+            dotL = Dot(color=DARK_GREY)
+            dotL.shift(2*RIGHT)
+            dotR = Dot(color=WHITE)
+            dotR.shift(2*LEFT )
+            
+            dotMiddle = VMobject().interpolate(dotL, dotR, alpha=0.3)
+            
+            self.add(dotL, dotR, dotMiddle)
+
+
+.. manim:: MoveAroundExample
+    
+    class MoveAroundExample(Scene):
+        def construct(self):
+            square = Square(color=BLUE,fill_opacity=1)
+            
+            self.play(square.shift,LEFT)
+            self.play(square.set_fill,ORANGE)
+            self.play(square.scale,0.3)
+            self.play(square.rotate,0.4)
+
+
+.. manim:: TextAlignementExample
+    :save_last_frame:
+    
+    class TextAlignementExample(Scene):
+          def construct(self):
+              title=PangoText("K-means clustering and Logistic Regression", color=WHITE)
+              title.scale_in_place(0.75)
+              self.add(title.to_edge(UP))
+              
+              t1=PangoText("1. Measuring").set_color(WHITE)
+              t1.next_to(ORIGIN,direction=RIGHT,aligned_edge=UP)
+              
+              t2=PangoText("2. Clustering").set_color(WHITE)
+              t2.next_to(t1,direction=DOWN,aligned_edge=LEFT)
+              
+              t3=PangoText("3. Regression").set_color(WHITE)
+              t3.next_to(t2,direction=DOWN,aligned_edge=LEFT)
+              
+              t4=PangoText("4. Prediction").set_color(WHITE)
+              t4.next_to(t3,direction=DOWN,aligned_edge=LEFT)
+            
+              x=VGroup(t1,t2,t3,t4).scale_in_place(0.7)
+              x.set_opacity(0.5)
+              x.submobjects[1].set_opacity(1)
+              self.add(x)
```

Comment:
We can move this one later to "Text and Latex".
Note, that this Pull request has not the most recent master, so 
https://manimce.readthedocs.io/en/latest/examples/tex.html
this section does not exist yet in this pr.
There are now two procedures:
1. (easy): we leave it like this and change this in a future pr
2. (more advanced): @MysaaJava : you go to your local manim folder, change the branch to  MysaaJava:newExamples and then you pull from the master "ManimCommunity:master".

Reply:
Well, it is more about «positions» than about «tex» in my opinion. What do others think ?

Reply:
yeah, good point.
I am also fine with leaving them there.

Reply:
So .... should i resolve this (it is a 50/50 with two opinions)

Reply:
Lets keep it here for now, there is #595 where we have to deal with the Text/Tex-section in general.